cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -409,6 +415,14 @@ body {
                     border-bottom: none;
                 }
             }
+
+            h3 {
+                margin-top: @grid_gutter-width / 2;
```

Comment:
Some additional funkiness happens on Sheerlike pages:

![screen shot 2017-07-26 at 15 24 16](https://user-images.githubusercontent.com/1044670/28639512-8e1b3136-7216-11e7-8d54-26aef2846977.png)

I think the on-demand assets may need to be recompiled, or something?